lib/config.go

```diff
@@ -96,6 +96,9 @@ type RootConfig struct {
 	// File-based locking is required when multiple users of lib are
 	// present on the same system
 	FileLocking bool `toml:"file_locking"`
+
+	// FileLockingPath specifies the path to use for the locking.
+	FileLockingPath bool `toml:"file_locking_path"`
```

Comment:
nit: can we fix this commit to not introduce this as bool and rewrite as string in next commit?

Reply:
absolutely, I thought I already did it, apparently I screwed the rebase.  Pushed a new version